README.md

```diff
@@ -1,83 +1,278 @@
-# AWS Load Balancer Controller Configuration
+# configuration-aws-lb-controller
```

Comment:
I liked the previous title more

Reply:
Is it a subjective preference? I just checked,e.g. https://github.com/upbound/configuration-aws-network , it looks like we do not have consistency in readme titles.